tests/end-to-end/upgrade/Gopkg.toml

```diff
@@ -17,19 +17,43 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.15.2"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.15.2"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.15.2"
+
+[[override]]
+  name="sigs.k8s.io/controller-runtime"
+  version = "v0.2.0"
 
 [[constraint]]
   name = "github.com/kyma-project/kyma"
-  revision = "7e6a7ca01ae5a402a3620ad227bca5c1d5638450"
+  version = "1.8.0"
+
+[[constraint]]
+  name = "github.com/michal-hudy/kyma"
```

Comment:
but after merge this will be reverted, right?

Reply:
Yes, of course :) Upgrade tests for `apicontroller` use tests from `backup` and we have a conflicts with dependencies on updating they to newer, so we must update firstly those dependencies in `backup`.